### PR TITLE
fixed long text contentful

### PR DESCRIPTION
**Pull request by humailhasankhan**

### CODE REVIEW

File: src/templates/singleCaseStudy.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ class CaseStudyTemplate extends React.Component {
     }
 
     const caseStudy = get(this.props, 'data.contentfulCaseStudies')
-
+    console.log(caseStudy)
```
Comment (pugoverflow) [inline src/templates/singleCaseStudy.js]: Take console log out please

File: src/templates/singleCaseStudy.js [hunk 2]
```diff
@@ -32,7 +32,14 @@ class CaseStudyTemplate extends React.Component {
             <h2 className="text-xl md:text-3xl">{caseStudy.title}</h2>
           </div>
           <div className="order-4 col-span-1 md:col-span-2 grow text-justify p-2 pt-6">
-            <p>{caseStudy.story.internal.content}</p>
+            {/* <p>{caseStudy.story.internal.content}</p> */}
```
Comment (pugoverflow) [inline src/templates/singleCaseStudy.js]: Take comment out please

**Review by pugoverflow — CHANGES_REQUESTED** [review]

**Comment by pugoverflow** [comment]
Good work here! Just make sure everyone understands the fix and how it works incase you are asked about it in your demo! :)

File: src/templates/singleCaseStudy.js (hunk 2 above)
Comment (humailhasankhan) [reply src/templates/singleCaseStudy.js]: It's done.

File: src/templates/singleCaseStudy.js (hunk 1 above)
Comment (humailhasankhan) [reply src/templates/singleCaseStudy.js]: That's done as well. Sorry didn't see this comment before.

**Review by maham511 — APPROVED** [review]

**Review by pugoverflow — APPROVED** [review]

**Pull request merged**